tests/cnf/core/network/security/tests/nftables-custom-firewall-test.go

```diff
@@ -107,11 +107,16 @@ var _ = Describe("nftables", Ordered, Label(tsparams.LabelNftablesTestCases), Co
 		By("Remove the static route to the external Pod network on each worker node")
 		addDeleteStaticRouteOnWorkerNodes(testPodList, routeMap, "del", hubIPv4Network)
 
+		By("Remove nftables entries from machineconfiguration nodeDisruptionPolicy")
+		removeMachineConfigurationNodeDisruptionPolicy()
+
+		By(fmt.Sprintf("Remove machine-configuration %s", mcNftablesName))
+		err := mco.NewMCBuilder(APIClient, mcNftablesName).Delete()
```

Comment:
Deleting the MachineConfig will sooner or later trigger a node reboot, no?
Just wondering if we have to explicitely reboot the node in disableNftablesIfActiveAndReboot or not. Is there a chance that the node gets rebooted twice? The test would still be valid, just an unnecessary reboot cycle. In any case, I'm fine with whatever you choose.